quinn-h3/src/server.rs

```diff
@@ -158,6 +391,61 @@ impl Connecting {
         tokio::spawn(ConnectionDriver(conn_ref.clone()));
         Ok((IncomingRequest(conn_ref), zerortt_accepted))
     }
+
+    /// Construct an HTTP/3 handsake future from an ongoing QUIC handshake.
+    ///
+    /// Makes supporting several protocols over one endpoint possible, by accepting
+    /// incoming connections initialization at the QUIC level. Client's ALPN can
+    /// then be used to route this new connection to the requested protocol. Say if
+    /// the ALPN matches [`quinn_h3::ALPN`], you can safely use [`Connecting::from_quic()`],
```

Comment:
This sentence should be reworded -- not exactly clear to me what you're trying to say here.